docs/using-the-python-driver/using-plugins/UsingTheLimitlessConnectionPlugin.md

```diff
@@ -0,0 +1,49 @@
+| `limitless_connection_max_retries_ms`                   | Integer |    No    | This property is the max number of retries the Limitless Connection Plugin will attempt when failing to connect to the database. During these retries, the plugin will attempt to connect to the least loaded transaction router that is available. If the max number of connection retries is exceeded, then the plugin will throw a `SQLException`. In this scenario, it is likely that the database is in an unhealthy state, and the `SQLException` should be caught and handled by your application.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | `5`           | `13`          |
+| `limitless_wait_for_transaction_router_info`            | Boolean |    No    | In scenarios such as application start-up, the cache of available transaction routers and their load metric info will be empty. If `limitlessWaitForTransactionRouterInfo` is set to `true`, the plugin will wait until the cache is populated before selecting a transaction router and connecting to it. This may be beneficial for applications that create a large number of connections on start-up, since these connections will be load-balanced. <br><br>Alternatively, if this property set to `false` and the cache is empty, the plugin will not wait for the cache to be populated and default to using the DB Shard Group endpoint to connect to until the cache is populated. This will result in connections being routed to a transaction router via Route 53 until the cache is populated. This may be beneficial for applications that prioritize quicker start-up times at the expense of some early connections not being load-balanced by the Limitless Connection Plugin. | `true`        | `false`       |
```

Comment:
Will need to replace occurrences of `true` and `false` with `True` and `False` for python